Source/WebCore/bindings/scripts/CodeGeneratorJS.pm

```diff
@@ -5529,6 +5530,17 @@ sub GenerateAttributeSetterBodyDefinition
         push(@$outputArray, "    asObject(valueToForwardTo)->methodTable()->put(asObject(valueToForwardTo), &lexicalGlobalObject, forwardId, value, slot);\n");
         push(@$outputArray, "    RETURN_IF_EXCEPTION(throwScope, false);\n");
         
+        push(@$outputArray, "    return true;\n");
+    } elsif ($attribute->extendedAttributes->{LegacyLenientSetter}) {
+        # https://webidl.spec.whatwg.org/#LegacyLenientSetter
+        assert("[LegacyLenientSetter] must be set with readonly") if !$attribute->isReadOnly;
+        assert("[LegacyLenientSetter] must not be with [PutForwards]") if $attribute->extendedAttributes->{PutForwards};
+        assert("[LegacyLenientSetter] must not be with [Replaceable]") if $attribute->extendedAttributes->{Replaceable};
+
+        push(@$outputArray, "    UNUSED_PARAM(lexicalGlobalObject);\n");
+        push(@$outputArray, "    UNUSED_PARAM(thisObject);\n");
+        push(@$outputArray, "    UNUSED_PARAM(value);\n");
+        push(@$outputArray, "    throwScope.assertNoException();\n");
```

Comment:
It would be even better not to define `throwScope` at all in this case, but this is acceptable.

Reply:
I addressed in https://github.com/WebKit/WebKit/pull/8121/commits/355eb9f362cced0af1226f160dfe521bf69a6f1b